src/cmd/acme/exec.c

```diff
@@ -243,6 +243,16 @@ execute(Text *t, uint aq0, uint aq1, int external, Text *argt)
 		free(r);
 		return;
 	}
+	/* Prefer Edit's <, |, > */
```

Comment:
This has an unintended side-effect.

With the original code, clicking with button 2 on something like `|ind` would set the selection to the output of the command. This version sets the selection to the empty string before that output.

I'll sort it out.

Reply:
FWIW, the side effect on dot in acme's `Edit |cmd` does not happen in sam. I'm inclined to regard acme's behavior as wrong and fix it, leaving this interpretation of `|cmd` alone.

(The new `Put` code should likely preserve dot.)

Reply:
/cc @rsc in case you have strong opinions.